plugins/confluence-plugin/src/index.html

```diff
@@ -0,0 +1,10 @@
+<html>
+  <head>
+    <script
+      defer
+      type="text/javascript"
+      src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.7/iframeResizer.contentWindow.js"
```

Comment:
I updated the cookiecutter template since this was created 🙂. This should now be
```suggestion
      src="https://cdn.jsdelivr.net/npm/@iframe-resizer/child"
```